devel/retdec/Portfile

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+categories          devel
+license             MIT
+platforms           darwin
+
+description         RetDec is a retargetable machine-code decompiler based on LLVM.
+
+long_description    RetDec is a retargetable machine-code decompiler based on LLVM.\
+                    The decompiler is not limited to any particular target architecture, \
+                    operating system, or executable file format.
+
+maintainers         nomaintainer
+
+github.setup        avast retdec 4.0 v
+
+revision            0
+
+checksums           rmd160  305e901b37441337d5c0dada1ccc5e737831ef1e \
+                    sha256  b0e3172d9d232899b71dac41446592c03abe089b8e9c7c1a9c8e8bfe92cf8b49 \
+                    size    8732003
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:cmake \
+                    port:doxygen \
```

Comment:
you do add the dependencies for building the documentation. It does appear though to do so you wiuld need to set: 

-DRETDEC_DOC=ON to build with API documentation (requires Doxygen and Graphviz, disabled by default)

Reply:
I can achive to generating control-flow graphs only when I enabled doxygen and graphviz ;)

I never thought that API documentation is nessesary for this tools to be hones.

Reply:
I didn't look carefully, only glanced over upstreams README. I assumed that you added the `doxygen` and `graphviz` dependencies to build the documentation, I don't see another reason to have them as a build dependency. If you need them for anything else at runtime, you should declare the appropriately.

Reply:
@reneeotten I need them as build dependency because if I build it without them => a part to generate control-flow graphs is escaped. I guess it is related to `graphviz` only and `doxygen` can be excluded. Let me test.

Reply:
You ware right => `doxygen` and `graphiz` aren't build dependency. In the fact that it needs only `graphviz` and to run.